### ISSUE TITLE
Link new wiki entries in README?

### ISSUE THREAD

**Issue by ghost** [AMOSus/amos-ss16-proj6#39]
In our README there is a link to some of the wiki entries on github. We have some new entries however, so we need to either
a) add them to the README
b) Just link to the wiki as a whole so we don't have to update every time

**Comment by danielreischl** [comment]
Yeah we should restructure the wiki and readme a bit. I would just add the links to the readme-file. Makes sense because I doubt that people will really go through the wiki if we just provide a link to the wiki.
